README.md

```diff
@@ -3,3 +3,15 @@
 The _Official_ Website of the Nevada County Free Code Camp.
 
 https://nevadacountyfreecodecamp.github.io/
+
+
+# Changes since May 30th
+Included normalize.css for style reset
+
+Went with Expletus and Raleway fonts as an idea, could be anything else though
+
+Wouldn't mind switching to SCSS and maybe something like bourbon/neat etc to practice something like that if other people are into it. Or not, and stick to basic CSS
```

Comment:
I forgot to update README.md, still learning how to use this;) If other people want to join that aren't comfortable with SCSS I could also switch back to CSS, though it's pretty basic SCSS. Do I just to another commit&push now after I update README.md? Or can I change it, amend the old commit and then push?